src/Symfony/Component/Translation/Bridge/Crowdin/CrowdinProvider.php

```diff
@@ -324,9 +348,7 @@ private function listStrings(int $fileId, int $limit, int $offset): array
         ]);
 
         if (200 !== $response->getStatusCode()) {
-            $this->logger->error(sprintf('Unable to list strings for file %d: "%s".', $fileId, $response->getContent()));
```

Comment:
`$throw` flag to `getContent` is true by default. So this warning cannot work with codes 3xx, 4xx, 5xx.
And only `200` is possible in 2xx (https://developer.crowdin.com/api/v2/#operation/api.projects.strings.getMany)